<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CDMetaPop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="QTable">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>671</width>
      <height>221</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>A</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>B</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>C</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>D</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>E</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>F</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>G</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>H</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>I</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>J</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>K</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>L</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>M</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>N</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>O</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>P</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Q</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>R</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>S</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>T</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>U</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>V</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>W</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>X</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Y</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Z</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>A</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>B</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>C</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>D</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>F</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>G</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>H</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>I</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>J</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>K</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>L</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>M</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>N</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>O</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>P</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Q</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>R</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>S</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>T</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>U</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>V</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>W</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>X</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Y</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Z</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="tryButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>PButton</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>671</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionRecent"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFullScreen"/>
    <addaction name="actionSplitScreen"/>
    <addaction name="actionInput"/>
    <addaction name="actionOutput"/>
    <addaction name="actionGraph"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuTAble">
     <property name="title">
      <string>Table</string>
     </property>
     <addaction name="actionSize"/>
     <addaction name="actionFont"/>
     <addaction name="actionType"/>
     <addaction name="actionHeader"/>
     <addaction name="actionFooter"/>
    </widget>
    <addaction name="menuTAble"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionCDMetaPop_About"/>
    <addaction name="actionFrequently_Asked_Questions"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="text">
    <string>FullScreen</string>
   </property>
  </action>
  <action name="actionSplitScreen">
   <property name="text">
    <string>SplitScreen</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>Input (PopOut)</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>Output (PopOut)</string>
   </property>
  </action>
  <action name="actionGraph">
   <property name="text">
    <string>Graph</string>
   </property>
  </action>
  <action name="actionRecent">
   <property name="text">
    <string>Recent</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionSize">
   <property name="text">
    <string>Size</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="actionType">
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionHeader">
   <property name="text">
    <string>Header</string>
   </property>
  </action>
  <action name="actionFooter">
   <property name="text">
    <string>Footer</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionCDMetaPop_About">
   <property name="text">
    <string>CDMetaPop About</string>
   </property>
  </action>
  <action name="actionFrequently_Asked_Questions">
   <property name="text">
    <string>Frequently Asked Questions</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
